README.md

```diff
@@ -143,4 +145,281 @@ If you want to support a new platform, please consider submitting a PR to get th
          target/ipk/helium-gateway-<version>-<platform>.<ipk or deb>
          ```
 
+## Additional usage info
+
+The Helium Gateway application can be configured to suit your hardware/software environment in a variety of ways - either from the command line, using customisations to the `settings.toml` file or with environment variables. The following sections describe this functionality in more detail as well as more general information on how to use the application.
+
+### Settings file
+
+The Helium Gateway application is configured using TOML files for your settings. The defaults can be found in the [default.toml](https://github.com/helium/gateway-rs/blob/main/config/default.toml) file in this repo. You should not edit this file directly, rather you should create a `settings.toml` file and store it either at the default expected locaton `/etc/helium_gateway/settings.toml` or at a custom location of your choosing by passing the `-c` flag to the `helium_gateway` application as shown below in the [general usage section](#general-usage-info).
+
+You can customise any of the fields shown in the [default.toml](https://github.com/helium/gateway-rs/blob/main/config/default.toml) file, however it is important to make sure that when editing you maintain the same ordering as shown in the default file. You do not need to include all fields in the `settings.toml` file - only the ones you want to change from default values - however maintaining the correct order is highly recommended to avoid any unexpected behaviour.
+
+For example, this config **will not work correctly** as it is in the wrong order:
+
+```
+[update]
+platform = "ramips_24kec"
+
+[log]
+method = "stdio"
+level = "info"
+timestamp = false
+
+region = "EU868"
+```
+
+Whereas this one will:
+
+```
+region = "EU868"
+
+[log]
+method = "stdio"
+level = "info"
+timestamp = false
+
+[update]
+platform = "ramips_24kec"
+```
+By maintaining the same layout as the `default.toml` file you can avoid any unexpected errors.
+
+The following are the settings that can be changed and their default and optional values:
+```
+keypair = "/etc/helium_gateway/gateway_key.bin" # can be any file location where you store the gateway_key.bin file
+listen_addr = "127.0.0.1:1680" # can be any ip address and port combination
+region = "US915" # possible values are : US915| EU868 | EU433 | CN470 | CN779 | AU915 | AS923_1 | AS923_2 | AS923_3 | AS923_4 | KR920 | IN865
+
+[log]
+method = "stdio" # either stdio or syslog
+level = "info" # possible values are: debug | info | warn
+timestamp = false # either true or false
+
+[update]
+enabled = true # either true or false
```

Comment:
Move the comments to the line before each setting?

Reply:
all sorted now